playbooks/documentation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Documentation
+description: How and where to document or share content
+---
+
+# Documentation
+
+As engineers we operate in many parallel forums: GitHub, Jira, Slack, Google docs, e-mail, Notion, and so on. What
+content should go where?
+
+## Documentation or discussion of a specific system
+
+When documentation relates to a particular system, leverage in-repository `README`s or `doc/`s wherever possible.
+This includes when the consumers may not exclusively be engineers.
+
+E.g.:
+
+- How to set-up a local development environment
+- Definitions of domain models
+- API documentation
+
+It _almost_ goes without saying, but other repo-specific interactions such as presenting the rationale for a design
```

Comment:
I’d recommend dropping the opening parenthetical statement, and starting this paragraph with “Other repo-specific ...”